core/arch/arm32/sta/sst_km_tests.c

```diff
@@ -0,0 +1,353 @@
+/*
```

Comment:
Right, I understand that it's a problem for API's not exposed. But in any case, this is functionality that is used by Trusted Applications (user TA), hence we should at least also have some tests in optee_tests that ensure that this functionality indeed is working. May I suggest that we keep this file for know, but that you also make a contribution to optee_test.git as soon as possible?